linera-service/src/proxy.rs

```diff
@@ -148,6 +152,13 @@ impl SimpleProxy {
         Ok(())
     }
 
+    pub fn start_metrics(address: &String) {
+        match address.parse::<SocketAddr>() {
+            Err(err) => error!("Invalid metrics address for {address}: {err}"),
```

Comment:
I wonder if the metrics config should be optional. If none is configured we don't call `start_metrics` anyway; but if it _is_ configured, we would return an error here and not start the proxy?

Reply:
Ah yes, this is supposed to `panic!`, the Server code already has that behavior. Is there a use case where we wouldn't want metrics enabled?

Reply:
I'm not sure; let's have the same behavior here as the server code for now.